golem-templates/templates/ts/ts-app-component-default/components-ts/component-name/src/main.ts

```diff
@@ -1,3 +1,4 @@
+/// <reference path="./generated/wit-generated.d.ts" />
```

Comment:
are this deliberatly added, or just accidentally added by some IDE?

Reply:
Deliberately added.
Typescript does not find the type definitions for me without explicitly including it.
(and doing it in the tsconfig.json via typeroots has the problem of disabling the default @types discovery in node_modules)